@here/harp-mapview/lib/MapView.ts

```diff
@@ -1675,6 +1680,20 @@ export class MapView extends THREE.EventDispatcher {
         return p;
     }
 
+    /**
+     * Returns a ray caster using the supplied screen positions.
+     *
+     * @param x The X position in css/client coordinates (without applied display ratio).
+     * @param y The Y position in css/client coordinates (without applied display ratio).
+     *
+     * @return Raycaster with origin at the camera and direction based on the supplied x / y screen
+     * points.
+     */
+    raycasterFromScreenPoint(x: number, y: number): THREE.Raycaster {
```

Comment:
Why do we need this method in the public interface?

Reply:
It is needed for the current implementation of the TerrainElevationProvider, because it calls raycaster.intersectObject on the TerrainTile mesh, however, when this implementation goes away, I can remove it.

Reply:
If this is temporary, please mark it as such (i.e. `@deprecated` or `@alpha`).
Actually we should mark the whole terrain API as `@alpha`. And the rest of the API as `@beta`.
See: https://api-extractor.com/pages/tsdoc/tag_alpha/